ingest-app/test/cmr/ingest/api/subscriptions_test.clj

```diff
@@ -3,15 +3,18 @@
    [clojure.string :as string]
    [clojure.test :refer :all]
    [cmr.ingest.api.core :as api-core]
-   [cmr.ingest.api.subscriptions :as subscriptions]))
+   [cmr.ingest.api.subscriptions :as subscriptions]
+   [cmr.transmit.metadata-db :as mdb]))
 
 (deftest ingest-subscription-test
```

Comment:
Need to add some api test error cases, i.e. POST with native-id, PUT without native-id.

Reply:
I think we want to allow POSt with native-id, so only PUT without native-id would be error.